config/src/config/quorum_store_config.rs

```diff
@@ -91,6 +91,8 @@ pub struct QuorumStoreConfig {
     pub num_workers_for_remote_batches: usize,
     pub batch_buckets: Vec<u64>,
     pub allow_batches_without_pos_in_proposal: bool,
+    /// Maximum number of transaction summaries stored in the proof queue.
+    pub max_txn_summaries_in_proof_queue: usize,
```

Comment:
Is it better to use u64 instead of usize ?
'QuorumStoreConfig' is being annotated with 'serialize' and 'deserialize'. So it is safer to use u64 instead of usize
(I see that there are other variables of usize, but we can be more cautious with new ones we add ? )

Reply:
Done.

Reply:
All the other integers in QuorumStoreConfig are `usize` for some reason. I replaced all the other occurrences of this config variable to u64.